apps/web/pages/settings/platform/oauth-clients/[clientId]/edit/index.tsx

```diff
@@ -29,6 +31,14 @@ export default function EditOAuthClient() {
   const router = useRouter();
   const params = useParams<{ clientId: string }>();
   const clientId = params?.clientId || "";
+
+  const { data: user, isLoading } = useMeQuery();
+  const { data: userBillingData, isPending: isUserBillingDataLoading } = useCheckTeamBilling(
```

Comment:
use isFetching

Reply:
> use isFetching

The `isPending` above is actually not being used so will completely remove that.